_apps/mason.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Mason
+layout: app
+cite: fu_mi_publications962
+contact: Manuel Holtgrewe
+category: official
+links:
+  download: http://packages.seqan.de/mason/
```

Comment:
```suggestion
  download: http://packages.seqan.de/mason2/
```

There are both mason and mason2.
I'd figure mason is the originally published version and everything after that is mason2.